packages/alpha-app/src/app.js

```diff
@@ -0,0 +1,153 @@
+const express = require("express");
+const path = require("path");
+const session = require("express-session");
+const { configureNunjucks } = require("./config/nunjucks");
+const {
+  validateOrganisationType,
+} = require("./journeys/organisationTypeService.js");
+const { validateHelpWithHint } = require("./journeys/helpWithHintService");
+const {
+  validateServiceDescription,
+} = require("./journeys/serviceDescriptionService");
+const { validateChooseLocation } = require("./journeys/chooseLocationService");
+const { validateEnterEmail } = require("./journeys/enterEmailService");
+const { validateFeedback } = require("./journeys/feedbackService");
+
+const crypto = require("crypto");
+const sessionId = crypto.randomBytes(16).toString("hex");
+const {
+  setGa4ContainerId,
+  setUaContainerId,
+  setStatusCode,
+  setTaxonomyValues,
+  setPageTitle,
+  setContentId,
+} = require("./config/gtmMiddleware.js");
+const { checkSessionAndRedirect } = require("./config/middleware");
+const i18next = require("i18next");
+const Backend = require("i18next-fs-backend");
+const i18nextMiddleware = require("i18next-http-middleware");
+const { i18nextConfigurationOptions } = require("./config/i18next");
+const { frontendVitalsInit } = require("@govuk-one-login/frontend-vital-signs");
+
+frontendVitalsInit(10000);
```

Comment:
And just this to amend to

frontendVitalsInit();

Reply:
Thanks, removed this value 👍